FusionIIIT/Fusion/settings/development.py

```diff
@@ -6,13 +6,23 @@
 
 ALLOWED_HOSTS = []
 
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.postgresql_psycopg2',
+#         'NAME': 'fusionlab',
+#         'HOST': '172.27.16.216',
+#         'USER': 'fusion_admin',
+#         'PASSWORD': 'hello123',
+#     }
+# }
+
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'fusionlab',
-        'HOST': '172.27.16.216',
-        'USER': 'fusion_admin',
-        'PASSWORD': 'hello123',
+        'HOST': 'localhost',
```

Comment:
Please don't commit or push this file. Remove it from the staging area before making any commit.